src/components/Home.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import AddNewBook from './AddNewBook';
+
+function Home() {
+  return (
+    <div className="homePage">
+      <AddNewBook />
+    </div>
+  );
+}
+
+export default Home;
```

Comment:
- [ ] There should be a component for rendering the list of books. I assume this is the component. Now, kindly import and render your create Book component here.